internal/resources/metal/organization_member/models.go

```diff
@@ -0,0 +1,40 @@
+package organizationmember
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+
+	"github.com/packethost/packngo"
+)
+
+type ResourceModel struct {
+	invitee         types.String `tfsdk:"invitee"`
+	invited_by      types.String `tfsdk:"invited_by"`
+	organization_id types.String `tfsdk:"organization_id"`
+	project_ids     types.Set    `tfsdk:"project_ids"`
+	nonce           types.String `tfsdk:"nonce"`
+	message         types.String `tfsdk:"message"`
+	created         types.String `tfsdk:"created"`
+	updated         types.String `tfsdk:"updated"`
+	roles           types.Set    `tfsdk:"roles"`
+	state           types.String `tfsdk:"updated"`
+}
+
+func (m *ResourceModel) parse(org *packngo.Invitation) diag.Diagnostics {
+	m.invitee = types.StringValue(org.Invitee)
+	m.invited_by = types.StringValue(org.InvitedBy.Href)
+	m.organization_id = types.StringValue(org.ID)
+
+	projectsList, _ := types.SetValueFrom(context.Background(), types.StringType, org.Projects)
+
+	m.project_ids = projectsList
+	m.nonce = types.StringValue(org.Nonce)
+	m.created = types.StringValue(org.CreatedAt.String())
+	m.updated = types.StringValue(org.UpdatedAt.String())
+	rolesList, _ := types.SetValueFrom(context.Background(), types.StringType, org.Projects)
```

Comment:
we need current context here like in https://github.com/equinix/terraform-provider-equinix/blob/12c5b6b21a6d385cca5863d6c58a5a3702b6a012/internal/resources/metal/connection/models.go#L101